docs/src/utils/index.ts

```diff
@@ -1,3 +1,13 @@
+import { DEFAULT_IMG_WIDTHS } from '../constants';
+
 export function cx(...classes: string[]) {
   return classes.filter(Boolean).join(' ');
 }
+
+export function getDefaultSrcSet({ src }: { src: string }) {
+  // When there's auth, get a list of the configured sizes for this org
+  // If they accept `*`, then fall back to this default
```

Comment:
This comment confused me a tad. 
Does `When there's auth` refer to the future, or what this function is currently doing?
I assume the former as I don't see any auth stuff. 

Perhaps "When auth becomes available from Keystatic cloud..."?

Or my personal fav: 
```
// TODO: When auth becomes available...
```
😂

Reply:
Haha you know, the last thing I did before I committed was remove the `TODO` 😂 

And yeah, you're absolutely right this refers to the future. How we integrate the config options (including the widths as picked up in this function) from the user's/project's/org's cloud storage will probably change or at least extend this. I actually now don't think we need the comment, just a sync on how cloud integration will work when we're up to that. That sound ok to you?

Reply:
Yup totally, there is a lot of unknown in how cloud images will integrate at this stage, so we can just come back to it then.